[2020-04-30 16:27:44,995] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.packerina.task.RunTestsTask.createTestJson(RunTestsTask.java:203)
	at org.ballerinalang.packerina.task.RunTestsTask.execute(RunTestsTask.java:135)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.TestCommand.execute(TestCommand.java:373)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
[2020-04-30 16:28:20,287] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.packerina.task.RunTestsTask.createTestJson(RunTestsTask.java:203)
	at org.ballerinalang.packerina.task.RunTestsTask.execute(RunTestsTask.java:135)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.TestCommand.execute(TestCommand.java:373)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
[2020-04-30 16:29:13,076] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.packerina.task.RunTestsTask.createTestJson(RunTestsTask.java:203)
	at org.ballerinalang.packerina.task.RunTestsTask.execute(RunTestsTask.java:135)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.TestCommand.execute(TestCommand.java:373)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
[2020-05-01 12:41:14,830] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 12:43:25,152] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 12:44:29,696] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 12:48:41,814] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.validateBuiltinTypeAnnotationAttachment(SemanticAnalyzer.java:2852)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.visit(SemanticAnalyzer.java:384)
	at org.wso2.ballerinalang.compiler.tree.BLangTypeDefinition.accept(BLangTypeDefinition.java:110)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.analyzeNode(SemanticAnalyzer.java:2536)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.analyzeNode(SemanticAnalyzer.java:2504)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.analyzeDef(SemanticAnalyzer.java:2496)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.visit(SemanticAnalyzer.java:251)
	at org.wso2.ballerinalang.compiler.tree.BLangPackage.accept(BLangPackage.java:161)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.SemanticAnalyzer.analyze(SemanticAnalyzer.java:225)
	at org.wso2.ballerinalang.compiler.CompilerDriver.typeCheck(CompilerDriver.java:270)
	at org.wso2.ballerinalang.compiler.CompilerDriver.compile(CompilerDriver.java:222)
	at org.wso2.ballerinalang.compiler.CompilerDriver.compilePackageSymbol(CompilerDriver.java:214)
	at org.wso2.ballerinalang.compiler.CompilerDriver.compilePackage(CompilerDriver.java:131)
	at org.wso2.ballerinalang.compiler.Compiler.compilePackages(Compiler.java:166)
	at org.wso2.ballerinalang.compiler.Compiler.compilePackage(Compiler.java:173)
	at org.wso2.ballerinalang.compiler.Compiler.compile(Compiler.java:91)
	at org.wso2.ballerinalang.compiler.Compiler.build(Compiler.java:99)
	at org.ballerinalang.packerina.task.CompileTask.execute(CompileTask.java:52)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.RunCommand.execute(RunCommand.java:284)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
[2020-05-01 12:50:18,482] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 13:03:00,301] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 13:04:07,958] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 13:05:29,486] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 13:06:09,818] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 13:08:22,117] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 13:16:19,529] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 14:01:52,044] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-01 17:33:55,155] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-02 11:59:57,989] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-02 11:59:58,142] SEVERE {b7a.log.crash} - Illegal group reference 
java.lang.IllegalArgumentException: Illegal group reference
	at java.util.regex.Matcher.appendReplacement(Matcher.java:857)
	at java.util.regex.Matcher.replaceAll(Matcher.java:955)
	at java.lang.String.replaceAll(String.java:2223)
	at org.ballerinalang.packerina.task.RunTestsTask.generateHtmlReport(RunTestsTask.java:262)
	at org.ballerinalang.packerina.task.RunTestsTask.execute(RunTestsTask.java:164)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.TestCommand.execute(TestCommand.java:373)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
[2020-05-02 12:01:02,590] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-02 12:02:20,534] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-02 12:06:09,345] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-07 14:07:46,636] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-07 14:07:46,732] SEVERE {b7a.log.crash} - Illegal group reference 
java.lang.IllegalArgumentException: Illegal group reference
	at java.util.regex.Matcher.appendReplacement(Matcher.java:857)
	at java.util.regex.Matcher.replaceAll(Matcher.java:955)
	at java.lang.String.replaceAll(String.java:2223)
	at org.ballerinalang.packerina.task.RunTestsTask.generateHtmlReport(RunTestsTask.java:262)
	at org.ballerinalang.packerina.task.RunTestsTask.execute(RunTestsTask.java:164)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.TestCommand.execute(TestCommand.java:373)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
[2020-05-07 14:15:26,946] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:53)
 
[2020-05-07 14:15:27,011] SEVERE {b7a.log.crash} - Illegal group reference 
java.lang.IllegalArgumentException: Illegal group reference
	at java.util.regex.Matcher.appendReplacement(Matcher.java:857)
	at java.util.regex.Matcher.replaceAll(Matcher.java:955)
	at java.lang.String.replaceAll(String.java:2223)
	at org.ballerinalang.packerina.task.RunTestsTask.generateHtmlReport(RunTestsTask.java:262)
	at org.ballerinalang.packerina.task.RunTestsTask.execute(RunTestsTask.java:164)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.TestCommand.execute(TestCommand.java:373)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
 
